ci-operator/step-registry/workers-rhel/pre-hook/sshtunnel/workers-rhel-pre-hook-sshtunnel-commands.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+
+export QE_SSH_PRIVATE_KEY=/var/run/qe/private_key
+echo "PLATFORM_VERSION: '${PLATFORM_VERSION}'"
```

Comment:
Here the `PLATFORM_VERSION` is pointing? vmc verson?

Reply:
It's rhel version, will add documentation for this ENV.

Reply:
Okay, get it, it is specified for rhel version, can we update the debug info a bit to make it easy understanding.

Reply:
sure